pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[project]
+name = "meilisearch"
+description = "The python client for Meilisearch API."
+license = { text = "MIT" }
+requires-python = ">=3.7"
+authors = [
+  { name = "Charlotte Vermandel", email = "charlotte@meilisearch.com" },
+]
+keywords = [
+  "search",
+  "python",
+  "meilisearch",
+]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+    "requests",
+    "camel-converter[pydantic]",
+    "black",
```

Comment:
I can’t seem to do a multi line selection on a phone so can’t do a normal suggestion here. Black and isort should be removed here and added as dev dependencies with pipenv.